spec/spec_helper.rb

```diff
@@ -114,4 +114,7 @@
   Kernel.srand config.seed
 =end
   config.include JSONAPI::RSpec
+
+  # Load shared examples
+  Dir['./spec/support/shared_examples/**/*.rb'].each { |f| require f }
```

Comment:
Are we all happy to store shared examples in this folder?

Reply:
Works for me. Generally I, and you may already agree with me, would prefer to avoid shared example, preferring custom matchers where possible. But I do agree here that it may be the better approach.